src/components/AdmissionsDash.js

```diff
@@ -111,16 +112,31 @@ export class AdmissionsDash extends React.Component {
     this.fetchStudents();
   };
 
-  changeStudentStage = (option) => {
-    this.value = { value: option.value, label: allStages[option.value] };
+  changeStudentStage = (selectedOption) => {
+    this.setState({
+      selectedOption,
+    });
+    console.log(selectedOption, "selectedOption");
     const { filterValues } = this.state;
-    if (option.value === "default") {
+    if (selectedOption === null) {
       this.stage = null;
       this.dataType = "softwareCourse";
       this.fetchStudents(filterValues);
       this.value = "Student Details";
     } else {
-      this.stage = option.value;
+      // const arr = [];
+      let arr = selectedOption.map((option) => {
+        return option.value;
+      });
+      console.log(arr, " i am arr");
+      if (arr.includes("default")) {
+        this.stage = null;
+      } else {
+        this.stage = selectedOption.map((option) => {
+          return option.value;
+        });
+      }
```

Comment:
Since the items in `this.stage` are the same as those in `arr` when `arr` doesn't include `"default"`, and since you don't need a copy, could you do something like this instead to shorten your code and avoid repetition:
```suggestion
      this.stage = arr.includes("default") ? null : arr;
```